serveradmin/backup.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import zipfile
+
+from datetime import datetime
+
+from django.conf import settings
+
+
+from ideasbox import __version__
+
+
+def make_name():
+    """Return backup formatted file name."""
+    return '{}.zip'.format('_'.join([
+        settings.IDEASBOX_ID,
+        __version__,
+        datetime.now().strftime(Backup.DATE_FORMAT)
+    ]))
+
+
+class Backup(object):
+
+    FORMAT = 'zip'
+    DATE_FORMAT = "%Y%m%d%H%M"
```

Comment:
[isoformat](https://docs.python.org/2/library/datetime.html#datetime.datetime.isoformat) looks good to me. I know that there are some issues with `:` within filenames on Windows though.